.github/workflows/post-apiview.yml

```diff
@@ -0,0 +1,28 @@
+name: After APIView
+
+on:
+  check_run:
+    types: [completed]
+    
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  event-handler:
+    name: After APIView
+    runs-on: ubuntu-24.04
+    if: ${{ github.event.check_run.check_suite.app.name == 'Azure Pipelines' && 
+          ( contains(github.event.check_run.name, 'APIView') ||
```

Comment:
Will we also need to add the spec-gen-sdk pipelines as well?

Reply:
It will need.
```suggestion
          ( contains(github.event.check_run.name, 'APIView') || contains(github.event.check_run.name, '(SDK Generation Job') || 
```